### PR TITLE
mothur screen.seqs bugfix

### PR DESCRIPTION
**Pull request by pmpcosta**
FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [X] - This PR does something else (explain below)

fixes https://github.com/galaxyproject/tools-iuc/issues/3288

Bugfix: $optimize is a multiple option parameter, and the options must be collated with dashes, not separated with commas. If multiple options are selected for the optimize parameter of screen.seqs, the job will crash.

### CODE REVIEW

**Review by bernt-matthias — CHANGES_REQUESTED** [review]
Could you add a test case for this?

**Comment by pmpcosta** [comment]
I didn't bother, as it is not a new parameter, but I will try, should be a good learning exercise..

File: tools/mothur/screen.seqs.xml [hunk 1]
```diff
@@ -176,6 +176,24 @@ echo 'screen.seqs(
             <param name="savelog" value="true"/>
             <expand macro="logfile-test"/>
         </test>
+        <test><!-- test for multiple optimize options minlenght, maxlength, maxhomop -->
+            <param name="fasta" value="Mock_S280_L001_R1_001_small.trim.contigs.fasta" ftype="fasta"/>
+            <param name="criteria" value="50"/>
+            <param name="optimize" value="minlength ,maxlength ,maxhomop"/>
```
Comment (bernt-matthias) [inline tools/mothur/screen.seqs.xml:182]: Cool. But I think the spaces befor the `,` are wrong.

Comment (pmpcosta) [reply tools/mothur/screen.seqs.xml:182]: Iwill check...

Comment (pmpcosta) [reply tools/mothur/screen.seqs.xml:182]: You are right, no spaces. Will see to it...

**Review by bgruening — APPROVED** [review]

**Review by bernt-matthias — APPROVED** [review]
Should we wait for a review of @shiltemann ?

**Comment by pmpcosta** [comment]
do we change @WRAPPER_VERSION@ to .1 ?

**Comment by bernt-matthias** [comment]
> do we change @WRAPPER_VERSION@ to .1 ?

Yep. That sounds good.

**Comment by bernt-matthias** [comment]
@pmpcosta anything left to do here? otherwise I would merge.

**Comment by pmpcosta** [comment]
No, I think it is done.

**Comment by bernt-matthias** [comment]
Thanks @pmpcosta

**Pull request merged**